src/components/staking/LiValidator.vue

```diff
@@ -35,9 +35,12 @@
         :alt="`validator logo for ` + validator.name"
       />
       <div class="validator-info">
-        <h3 class="li-validator-name">
+        <h3 v-if="!validator.name.startsWith('0x')" class="li-validator-name">
           {{ validator.name }}
         </h3>
+        <h3 v-else>
+          <Bech32 :address="validator.name" />
```

Comment:
this shouldn't be used for validator names

Reply:
Ok, another misunderstanding :sweat_smile: